pkg/agent/metrics/prometheus.go

```diff
@@ -94,6 +94,30 @@ var (
 		},
 		[]string{"operation"},
 	)
+
+	TotalConnectionsInConnTrackTable = metrics.NewGauge(
+		&metrics.GaugeOpts{
+			Name:           "antrea_agent_conntrack_total_connection_count",
+			Help:           "Number of connections in the conntrack table, update every flowPollInterval.",
```

Comment:
suggestion: Number of connections in the conntrack table. This metric gets updated at an interval specified by flowPollInterval, a configMap parameter in antrea-agent.conf.

Reply:
Thanks, done.